app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -219,6 +181,53 @@ def import_google_students
 
   private
 
+  def set_classroom_variables
+    @tab = params[:tab] #|| "manageUnits"
```

Comment:
lets get rid of the unreachable code in the comment.

Reply:
Sure